lib/generators/manageiq/plugin/templates/.gitignore

```diff
@@ -2,7 +2,6 @@
 /.yardoc
 /Gemfile.lock
 /_yardoc/
-/config/secrets.yml
```

Comment:
New plugins should not have existing secrets to worry about accidentally committing. Additionally, rails credentials aren't stored in the plugin directory, so we only need to .gitignore in the manageiq application.